status-group-chats-spec.md

```diff
@@ -28,177 +28,123 @@ This documents describes the group chat protocol used by the status application.
 
 ## Membership updates
 
-Membership updates messages are used to propagate group chat membership changes. The transit format is described in the [Status Payload Specs](status-payload-specs.md). Here we will be describing each specific field.
-
-The format is:
+Membership updates messages are used to propagate group chat membership changes. The protobuf format is described in the [Status Payload Specs](status-payload-specs.md). Here we will be describing each specific field.
+
+The protobuf messages are:
+
+```protobuf
+// MembershipUpdateMessage is a message used to propagate information
+// about group membership changes.
+message MembershipUpdateMessage {
+  // The chat id of the private group chat
+  string chat_id = 1;
+  // A list of events for this group chat, first 65 bytes are the signature, then is a 
```

Comment:
We sign the raw bytes, so those need to be sent over and easily extracted.
So your suggestion would be:

```
message Event {
 repeated bytes signature = 1;
 repeated bytes payload = 2; // This is actually a MembershipUpdateEvent
}
```

which works fine, and it is more explicit.

I slightly prefer the concatenation mainly for ease of use, clients will take the event and store it as it, unchanged, so it can be relayed to third party, and probably save a few bytes.

Either though are perfectly fine I think.

Reply:
@cammellos ok that makes sense, lets keep it as is then.